Document/0x06b-Basic-Security-Testing.md

```diff
@@ -11,13 +11,19 @@ Although you can use a Linux or Windows machine for testing, you'll find that ma
 The following is the most basic iOS app testing setup:
 
 - ideally macOS machine with admin rights
-- Wi-Fi network that permits client-to-client traffic or USB multiplexing
+- Wi-Fi network that permits client-to-client traffic
 - at least one jailbroken iOS device (of the desired iOS version)
 - Burp Suite or other interception proxy tool
 
-#### Setting up the iOS SDK
+#### Setting up Xcode and Command Line Tools
 
--- ToDo: https://github.com/OWASP/owasp-mstg/issues/1242
+Xcode is an Integrated Development Environment (IDE) for macOS that contains a suite of tools developed by Apple for developing software for macOS, iOS, watchOS, and tvOS. You can [download Xcdoe for free from the official Apple website](https://developer.apple.com/xcode/ide/ "Apple Xcode IDE"). Xcode will offer you different tools and functions to interact with an iOS device that can be helpful during a penetration test, such as monitoring, analyzing logs or sideloading apps.
```

Comment:
- I'd write "Xcode offers different tools ..." 
- typo: You can [download **Xcode** for free from the official Apple website]
- do you mean "file access/network monitoring"?

Reply:
I removed monitoring, fixed the typo and shortened the first sentence a bit.